### PR TITLE
Scheudled post show more

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Added 'Show More' option to long draft and scheduled posts.

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-61210

#### Screenshots
Before-
Screenshot 2024-11-12 at 7 27 41 AM

After-
Screenshot 2024-11-12 at 7 31 11 AM

#### Release Note
```release-note
none
```

### CODE REVIEW

**Review by pvev — APPROVED** [review]
LGTM! 🎉

File: webapp/channels/src/components/drafts/panel/panel_body.tsx [hunk 1]
```diff
@@ -66,37 +67,43 @@ function PanelBody({
...
-                        />
-                    )}
-                </div>
+                </ShowMore>
+                {(fileInfos.length > 0 || uploadsInProgress?.length > 0) && (
+                    <FilePreview
+                        fileInfos={fileInfos}
+                        uploadsInProgress={uploadsInProgress}
+                    />
+                )}
```
Comment (larkox) [inline webapp/channels/src/components/drafts/panel/panel_body.tsx:106]: Just to verify. This no longer lives inside the `DraftPanelBody__right`, and I don't see any relevant css changes about that.

All css that was supposed to be applied to this before is still being applied? (the one that comes to mind more easily is highlights, but not sure if anything else)

**Review by larkox — APPROVED** [review]
LGTM, just one thing to verify.

File: webapp/channels/src/components/drafts/panel/panel_body.tsx (hunk 1 above)
Comment (harshilsharma63) [reply webapp/channels/src/components/drafts/panel/panel_body.tsx:106]: yes, all css is still valid and looks perfect 👍

**Review by asaadmahmood — APPROVED** [review]

**Pull request merged**